README.md

```diff
@@ -144,6 +146,7 @@ See also [Static Site Generators](#static-site-generators) and  [Content Managem
   * [Jitsi Meet](https://jitsi.org/Projects/JitsiMeet) - Jitsi Meet is an OpenSource (MIT) WebRTC JavaScript application that uses Jitsi Videobridge to provide high quality, scalable video conferences. `MIT ` `JavaScript`
   * [Jitsi Video Bridge](https://jitsi.org/Projects/JitsiVideobridge) - WebRTC compatible Selective Forwarding Unit (SFU) that allows for multiuser video communication. `Apache` `Java`  
   * [Kandan](http://getkandan.com/) – Kandan is an Open Source Alternative to HipChat. - `AGPL` 
+  * [Matrix](https://matrix.org) - An open standard for decentralized persistent communication (reference implementation: `Apache` `Python`)
```

Comment:
I'd prefer if you could link to separate server and web client pages for Matrix, like:

 * [Synapse](https://matrix.org/blog/project/synapse/) - A server for [Matrix](https://matrix.org), an open standard for decentralized persistent communication ([Source code](https://github.com/matrix-org/synapse)) `Apache` `Python`
  * [Matrix console web](https://matrix.org/blog/project/matrix-console-web/) ([Source code](https://github.com/matrix-org/matrix-angular-sdk)) - A web client meant to be a showcase of Matrix capabilities, and reference implementation of the Matrix standard`Apache` `Javascript`

Asking this because we don't reference standards and protocols, just applications.